### ISSUE TITLE
Inability to import curl command

### ISSUE THREAD

**Issue by stkrzysiak** [a85/POSTMan-Chrome-Extension#1157]
Hi, 
I am trying to import a curl command that looks like this:

```
curl "http://qa-demo.example.com/shop/api/loginCredentials" -H "Origin: http://qa-demo.example.com" -H "Accept-Encoding: gzip, deflate" -H "Accept-Language: en-US,en;q=0.8" -H "User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/43.0.2357.124 Safari/537.36" -H "Content-Type: application/vnd.example.com+emberv3.json" -H "Accept: application/vnd.example.com+emberv3.json" -H "Referer: http://qa-demo.example.com/shop/app/" -H "X-Requested-With: XMLHttpRequest" -H "Connection: keep-alive" -H "X-FirePHP-Version: 0.0.6" --data-binary "{""loginCredential"":{""_id"":""2052"",""username"":""jerrymguire"",""password"":""password"",""success"":false,""errorMessage"":null,""passwordExpired"":false,""application"":null,""authToken"":null,""consumerContextId"":0}}" --compressed
```

Yet I get "Errow while importing Curl: undefined".  As far as I can tell, there is nothing in the dev console during this import process.

I'd appreciate any insight, thank you!

**Comment by kriserickson** [comment]
I'm seeing the same thing.  Would love to be able to import curl exported from the network panel, but I can't even get simple Curls to work.

**Comment by stkrzysiak** [comment]
@kriserickson I am not sure what you consider simple, but I can verify that importing other curls works just fine for me.  This example search on youtube for 'postman chrome' imports without an issue:

```
curl "https://www.youtube.com/results?search_query=postman+chrome&spf=navigate" -H "X-SPF-Referer: https://www.youtube.com/" -H "Referer: https://www.youtube.com/" -H "X-SPF-Previous: https://www.youtube.com/" -H "X-FirePHP-Version: 0.0.6" -H "User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/43.0.2357.124 Safari/537.36" --compressed
```

EDIT: Accidently pasted content twice.

**Comment by kriserickson** [comment]
Sorry I was being very rude and not very helpful, simple is not the correct way to put it (it appears all the things I was debugging had encoded values in either the cookie or the data).  It appears that  as soon as it includes encoded data it fails, so:

```
curl "https://empressa.kris1.mha.local/global/agents/json_service.html" --data "method=json.log&logs="%"5B"%"7B"%"22date"%"22"%"3A"%"222015-06-17T17"%"3A24"%"3A56.945Z"%"22"%"2C"%"22type"%"22"%"3A"%"22info"%"22"%"2C"%"22message"%"22"%"3A"%"22Method"%"3A+https"%"3A"%"2F"%"2Fempressa.kris1.mha.local"%"2Fglobal"%"2Fagents"%"2Fjson_service.html"%"3Fmethod"%"3Dcontextmenu.getSubmenu"%"26session_key"%"3D47r4qi0m37penok5r1pf8s4266"%"26validate_abilities"%"255Btheme_editor"%"255D"%"3Dsf.settings"%"26validate_abilities"%"255Bedit_page"%"255D"%"3Dsf.settings"%"26validate_abilities"%"255Borders"%"255D"%"3Dsf.orders+took+208+ms"%"22"%"7D"%"5D" --compressed
```

fails but

```
curl "https://empressa.kris1.mha.local/global/agents/json_service.html" --data "method=json.log" --compressed
```

works.   And:

```
curl "https://empressa.kris1.mha.local/global/agents/json_service.html" -H "Cookie: _pk_id.2.0ad2=3937c414e673921f.1399937976.1.1399938176.1399937976.; _pk_id.6.0ad2=a988658e30d5c9ce.1415315962.1.1415315962.1415315962.; _pk_id.4.0ad2=37f0f935b9bb8fd4.1425685070.1.1425685192.1425685070.; monster=0"%"2Fg1sixviX0xrJxJ3ZFg4pgrcYEHsLubPQJVqcfVDdy8V3gtMyD"%"2BdQ"%"3D"%"3D; nextPageStartTime=1434561892608; " -H "Connection: keep-alive" -H "Referer: https://empressa.kris1.mha.local/Cart"  --compressed
```

Fails but

```
curl "https://empressa.kris1.mha.local/global/agents/json_service.html" -H "Cookie: _pk_id.2.0ad2=3937c414e673921f.1399937976.1.1399938176.1399937976.; _pk_id.6.0ad2=a988658e30d5c9ce.1415315962.1.1415315962.1415315962.; _pk_id.4.0ad2=37f0f935b9bb8fd4.1425685070.1.1425685192.1425685070.; nextPageStartTime=1434561892608; " -H "Connection: keep-alive" -H "Referer: https://empressa.kris1.mha.local/Cart"  --compressed
```

works.  Now that I figured this out, I can use the curl import and just remove the encoded data to get it to work.

**Comment by stkrzysiak** [comment]
It appears that this error is no longer thrown however the POST still fails for me.  I think at the time I was using Firefox firbug's 'copy as curl' and today I tried chrome's dev tools 'copy as curl(bash)' option which worked great.  I'm not certain if this should remain a bug or not, I'll leave that to the maintainers.

**Comment by stkrzysiak** [comment]
I no longer see this error when importing, but I still can not successfully POST using this mechanism.  I am fairly certain it has to do with the JSON data in the POST.  I'm just going to rely on curl for now.

**Comment by abhijitkane** [comment]
@stkrzysiak This has been fixed in the latest version (4.4.2) of the app. To update - https://www.getpostman.com/docs/updating
